public/app/themes/justice/functions.php

```diff
@@ -96,6 +97,9 @@
 $taxonomies = new Justice\Taxonomies();
 $taxonomies->addHooks();
 
+$templates = new Justice\Templates();
+$templates->addHooks();
```

Comment:
This will create `$templates` in the global scope. Are we sure nothing will overwrite this later, during PHP execution?